File: testsuite/matlab_and_octave/util/fix_distance_name_inconsistency.m
function y = fix_distance_name_inconsistency (dname)
	dname=upper(dname);
	if findstr('WORDDISTANCE', dname)
		pos=findstr('WORDDISTANCE', dname);
		y=dname(1:pos-1);
	elseif findstr('DISTANCE', dname)
		pos=findstr('DISTANCE', dname);
		y=dname(1:pos-1);
	elseif findstr('METRIC', dname)
		pos=findstr('METRIC', dname);
		y=dname(1:pos-1);
	else
		y=dname;
	end

File: tests/regression/r_static/util/fix_distance_name_inconsistency.R
fix_distance_name_inconsistency <- function(dname) {
	dname=toupper(dname)
	if (regexpr('WORDDISTANCE', dname)>0) {
		return(strsplit(dname, 'WORDDISTANCE')[[1]])
	} else if (regexpr('DISTANCE', dname)>0) {
		return(strsplit(dname, 'DISTANCE')[[1]])
	} else if (regexpr('METRIC', dname)>0) {
		return(strsplit(dname, 'METRIC')[[1]])
	} else {
		return(dname);
	}
}
